### PR TITLE
Bump Bootstrap, AdminLTE & FontAwesome

### PR DESCRIPTION
**Pull request by samiashi**
- Bump Bootstrap to v5
- Bump AdminLTE to v3.2.0
- Bump FontAwesome to v6.5.1
- Bump Bootswatch to v5.3.3

Fixes https://github.com/farridav/django-jazzmin/issues/547

### CODE REVIEW

**Comment by farridav** [comment]
Nice, this is on my radar, though it will need checking through visually to make sure it all behaves nicely and still looks jazzy

File: README.md [hunk 1]
```diff
@@ -21,7 +21,7 @@ them as quickly as I would like, but im trying to get through them all now, hope
 ![Django Versions](https://img.shields.io/badge/django-%3E%3D2-brightgreen)
 [![Coverage Status](https://coveralls.io/repos/github/farridav/django-jazzmin/badge.svg?branch=master)](https://coveralls.io/github/farridav/django-jazzmin?branch=master)
 
-Drop-in theme for django admin, that utilises AdminLTE 3 & Bootstrap 4 to make yo' admin look jazzy
+Drop-in theme for django admin, that utilises AdminLTE 3 & Bootstrap 5 to make yo' admin look jazzy
```
Comment (PacificGilly) [inline README.md:24]:
```suggestion
Drop-in theme for django admin, that utilises AdminLTE 3.2 & Bootstrap 5 to make yo' admin look jazzy
```

**Review by PacificGilly — COMMENTED** [review]
Nice PR. Do you mind QAing this too, please?

**Comment by jameslao** [comment]
AdminLTE 3.2 is indeed a drop-in replacement but it's not targeting bootstrap 5 in itself... I remember having some issues with tabs when I upgraded bootstrap. That's why I tried to work with AdminLTE 4 even though it's in alpha. I'll submit a separate PR.

**Comment by abe-101** [comment]
Any update on this?

**Comment by SourceDoctor** [comment]
would appreciate this change

**Comment by samiashi** [comment]
@farridav @jameslao @PacificGilly any updates on this? Would be good to move this forward

**Comment by PacificGilly** [comment]
> @farridav @jameslao @PacificGilly any updates on this? Would be good to move this forward

@samiashi Would you mind QAing this please with the test app? 🙂

**Comment by jucajuca** [comment]
I could be wrong, but I think this merge request still needs to update the bootswatch themes.  
https://bootswatch.com/

**Comment by samiashi** [comment]
Screenshot 2024-11-25 at 5 26 17 PM
Screenshot 2024-11-25 at 5 26 33 PM
Screenshot 2024-11-25 at 5 26 57 PM

**Review by PacificGilly — APPROVED** [review]
LGTM

**Comment by samiashi** [comment]
@PacificGilly sorry it took this long!

**Comment by jucajuca** [comment]
@farridav any plans to release this?

**Comment by farridav** [comment]
I hadn't realised it was unreleased, let me get a release out today, thanks for the reminder

**Comment by benjaggard-holloway** [comment]
@farridav Is there any chance a release could be made for this please? 

I have a pentesting audit that is flagging the out of date bootstrap for our site that could really do with this.

**Comment by farridav** [comment]
Sure, I'll have this done today

**Pull request merged**